osgeo_importer/forms.py

```diff
@@ -1,9 +1,81 @@
+import os
 from django import forms
 from .models import UploadFile
+import zipfile
+import tempfile
 
 
-class UploadFileForm(forms.ModelForm):
+class XXXUploadFileForm(forms.ModelForm):
 
     class Meta:
         model = UploadFile
         fields = ['file']
+
+class UploadFileForm(forms.Form):
+    file = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}))
```

Comment:
it's an attribute so we can do it, and this isn't new to importer, but `file` is a builtin name we're shadowing